js/app/modules/banner/title.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2016 Endless Mobile, Inc.
+
+const GObject = imports.gi.GObject;
+const Gtk = imports.gi.Gtk;
+
+const Module = imports.app.interfaces.module;
+const Utils = imports.app.utils;
+
+/**
+ * Class: Title
+ *
+ * A module for the application's title.
+ *
+ */
+const Title = new Module.Class({
+    Name: 'Banner.Title',
+    Extends: Gtk.Grid,
+
+    Properties: {
+        /**
+         * Property: show-subtitle
+         * Whether to show an application subtitle underneath the image.
+         * Subtitle will be taken from the desktop file description field.
+         */
+        'show-subtitle': GObject.ParamSpec.boolean('show-subtitle',
+            'Show Subtitle', 'Show Subtitle',
+            GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
+            false),
+    },
+
+    Template: 'resource:///com/endlessm/knowledge/data/widgets/banner/title.ui',
+    InternalChildren: [ 'title-label', 'subtitle-label' ],
+
+    _init: function (props={}) {
+        this.parent(props);
+
+        let app_info = Utils.get_desktop_app_info();
+
+        let title = app_info.get_display_name();
+        this._title_label.label = title;
```

Comment:
Maybe some sync up with dynamic logo in order here? Is there ever a reason to show something different in this text then with our logo system text? @tchx84 @rmacqueen ?